app/helpers/feeder_form_builder.rb

```diff
@@ -51,19 +62,13 @@ def check_box(method, options = {})
   end
 
   def date_field(method, options = {})
-    value = options[:value] || object&.public_send(method)
-    options[:value] = value.try(:strftime, "%Y-%m-%d") || value
-
     add_flatpickr_controller(options)
     text_field(method, options)
   end
 
-  def time_field(method, options = {})
-    value = options[:value] || object&.public_send(method)
-    options[:value] = value.try(:strftime, "%Y-%m-%d %-l:%M:%S %p %Z") || value
-    add_data(options, :timestamp, true)
-    add_flatpickr_controller(options)
-    text_field(method, options)
+  def time_zone_field(method, options = {})
+    add_data(options, :controller, TIME_ZONE_CONTROLLER)
+    select method, IMPORTANT_ZONES, {selected: "UTC"}, options
```

Comment:
will we ever need other time zones as options here? or is it generally assumed that the selected timezone will either be a US time zone, and anything else that doesn't gets the `"UTC"` label? (unless i'm reading this `select` component incorrectly.)

Reply:
also, was the `time_field` component removed out of here completely? i guess it makes sense since you would parse time off of a given value in the `time_field`, but that's not until a later step. first step is selecting a time; the time is used as an argument with a `zone.parse()` method in order to correctly give the full date and time, which is what we are trying to save in the db

Reply:
We could certainly add more, but we don't need to for this to work.  UTC is rendered server-side, but we change what's selected in JS.

If you look at the `time_zone_controller.js`, it handles this dropdown.  If it detects you're in one of these "important zones", it selects it.  Otherwise, it _adds a new option_ for your timezone string and selects that.  So should be compatible with any zone (I hope).

Yeah, I removed `time_field` entirely for now (it greatly simplifies the `flatpickr_controller.js`).  And the `dropdate_picker_controller.js` became its own thing - coordinating the date/time fields.  I could see making that a more generic component, if we find we want to reuse it somewhere else.